### PR TITLE
Refine the document of Metering & Billing in Chinese

### PR DESCRIPTION
**Pull request by yunkunrao**
Signed-off-by: Rao Yunkun <yunkunrao@yunify.com>

### CODE REVIEW

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md [hunk 1]
```diff
@@ -5,3 +5,39 @@ description: "启用 KubeSphere 中的计费功能，查看一段时间内您资
 linkTitle: "启用计费"
 weight: 15420
 ---
+
+## 开启计费功能
+
+默认计费功能是关闭的，可以通过以下步骤开启计费功能：
```
Comment (LinuxSuRen) [inline content/zh/docs/toolbox/metering-and-billing/enable-billing.md:11]: According to https://github.com/kubesphere/ks-installer/pull/1409 and https://github.com/kubesphere/ks-installer/pull/1406 , this feature is enabled by default. So I guess you are just given an opposite description.

And I would recommend that telling users about the ClusterConfigration.

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md [hunk 2]
```diff
@@ -5,3 +5,39 @@ description: "启用 KubeSphere 中的计费功能，查看一段时间内您资
...
+
+1. 在 configmap `ks-metering-config` 中指定不同资源类型的价格（初始为-1）
```
Comment (LinuxSuRen) [inline content/zh/docs/toolbox/metering-and-billing/enable-billing.md:13]: `ConfigMap` is better than `configmap`

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md [hunk 3]
```diff
@@ -5,3 +5,39 @@ description: "启用 KubeSphere 中的计费功能，查看一段时间内您资
...
+        ingressNetworkTrafficPerMegabytesPerHour: 1
+        egressNetworkTrafficPerMegabytesPerHour: 1
+        pvcPerGigabytesPerHour: 2.1
+kind: ConfigMap
+metadata:
+  name: ks-metering-config
+  namespace: kubesphere-system
+```
+
+> 上面定义了包含货币单位在内的价格信息：货币目前支持人民币 (CNY) 和美元 (USD) ，举例来说，上面配置的含义是 CPU 每 core 每小时 1.5 美元，内存每 GB 每小时 5 美元， ingress 网络流量每 MB 每小时 1 美元， egress 网络流量每 MB 每小时 1 美元， PVC 每 GB 每小时 2.1 美元。
```
Comment (LinuxSuRen) [inline content/zh/docs/toolbox/metering-and-billing/enable-billing.md:37]: What happened if a user gives an unsupported currency?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for providing the documentation about the `Metering and Billing` part. I left some comments.

**Comment by LinuxSuRen** [comment]
/retitle Refine the document of Metering & Billing in Chinese

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md (hunk 1 above)
Comment (yunkunrao) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md:11]: In fact metering is enabled by default, but the other part billing was disabled by by default with invalid resource price -1. In other words, with default installation metering is enabled and billing is disabled. User can get billing info by modify metering ConfigMap.

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md (hunk 2 above)
Comment (yunkunrao) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md:13]: got it

File: content/zh/docs/toolbox/metering-and-billing/enable-billing.md (hunk 3 above)
Comment (yunkunrao) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md:37]: Yeah, currently there's no currency unit check. I will discuss this part with @harrisonliu5

Comment (LinuxSuRen) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md:37]: What I want to say is that we can tell users what happened if they provide an unsupported currency. For example:

* Use a default currency
* Or show tips to let users know they just provide an unsupported currency

Comment (yunkunrao) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md:37]: If user set other currency unit, console will display "CNY".

Comment (LinuxSuRen) [reply content/zh/docs/toolbox/metering-and-billing/enable-billing.md]: Got it.

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**